modules/openapi-generator/src/main/resources/python/model_utils.mustache

```diff
@@ -1217,7 +1217,7 @@ def model_to_dict(model_instance, serialize=True):
                 elif isinstance(value[0], ModelSimple):
                     result[attr] = [x.value for x in value]
                 else:
-                    result[attr] = [model_to_dict(x, serialize=serialize) for x in value]
+                    result[attr] = [model_to_dict(x, serialize=serialize) if hasattr(x, '_data_store') else x for x in value]
```

Comment:
How about adding a test that will fail if this is broken? If that had existed then this bug would not have been introduced. What do you think?

Reply:
Sure. Will add it